osquery/table/corefoundation_helpers_darwin.go

```diff
@@ -0,0 +1,76 @@
+package table
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#include <CoreFoundation/CoreFoundation.h>
+*/
+import "C"
+import (
+	"fmt"
+	"reflect"
+	"unsafe"
+)
+
+func copyValue(key, domain, username string) C.CFPropertyListRef {
+	return C.CFPreferencesCopyValue(
```

Comment:
Do we need an associated `CFRelease` with this? https://developer.apple.com/library/archive/documentation/CoreFoundation/Conceptual/CFMemoryMgmt/Concepts/Ownership.html#//apple_ref/doc/uid/20001148-103029